tutorials/README.md

```diff
@@ -0,0 +1,168 @@
+# Tutorials for using the `sbi` toolbox
+
+These `sbi` tutorials are aimed at two sepatate groups
+1.  _users,_ e.g., domain scientists that aim to get an introduction to the method to then apply it to their (mechanistic) models
```

Comment:
Move the comma behind the underscore, no?

Reply:
will do